tests/api.c

```diff
@@ -13117,8 +13118,8 @@ static int test_wc_AesCbcEncryptDecrypt (void)
                                                     iv, AES_DECRYPTION);
         }
         if (ret == 0) {
-            ret = wc_AesCbcDecrypt(&aes, dec, enc, AES_BLOCK_SIZE);
-            if (ret != 0 || XMEMCMP(vector, dec, AES_BLOCK_SIZE) != 0) {
+            ret = wc_AesCbcDecrypt(&aes, dec, enc, sizeof vector);
```

Comment:
sizeof(vector)

Reply:
I like to remind myself that `sizeof` isn't a function, but it seems to distract a lot of other devs.  changed to your way.